migrations.lock

```diff
@@ -390,6 +390,8 @@ domain
  0012_operatorcalllimitsettings
  0013_accountconfirmationsettings_squashed_0016_alter_smsaccountconfirmationsettings_project_name (4 squashed migrations)
  0014_appreleasemodesetting
+ 0015_domainsettings
+ 0016_auto_20240419_2223
```

Comment:
Can these migrations be squashed? If not, would be nice if the second one had a nicer name.